state-chain/runtime/src/chainflip/vault_swaps.rs

```diff
@@ -95,7 +95,7 @@ pub fn bitcoin_vault_swap(
 
 	Ok(VaultSwapDetails::Bitcoin {
 		nulldata_payload: encode_swap_params_in_nulldata_payload(params),
-		deposit_address: derive_btc_vault_deposit_address(private_channel_id),
+		deposit_address: derive_btc_vault_deposit_addresses(private_channel_id).1,
```

Comment:
Maybe there's a better way to ensure that this is correct? Right now I have to read the implementation of `derive_btc_vault_deposit_addresses` to know what the `.1` represents. 

*edit: see my other comment re. returning a struct instead of a tuple*